### DISCUSSION TITLE
Ethernet transfer speed between computers

### DISCUSSION THREAD

**Discussion by Rccrampton** [apohl79/audiogridder#935] [Q&A]
I have Audiogridder seemingly working well between 2 older computers. Both have Gigabit Ethernet onboard but I'm getting a 12 Mbps transfer speed between the two. It seems simple data transfer (files or folders) transfer at near 100 Mbps but Audiogridder still is much slower. Is this normal?

**Comment by apohl79** [comment]
When you say slower, do you mean AG does not use the available bandwidth and you get issues during live playback? If you would do an offline bounce, the DAW is passing audio data as fast as possible, and AG should use as much bandwidth as it can get and the server/plugins can process.

**Reply by Rccrampton** [reply]
If I look at Windows task manager ethernet send/receive during DAW operation with Audiogridder running, the maximum transfer I get is around 12 Mbps when my adapter is rated at 100 Mbps. Maybe this is normal as audiogridder seems to be working ok. I just thought that since my adapter is 100 Mbps the ethernet transfer between computers should be faster. This on the other hand may be perfectly normal.

**Reply by Kjudkins** [reply]
That is perfectly normal depending on how many tracks you're sending. 
If I have no tracks armed from VEPRO for instance, I'm receiving 6mb/s
If I have 8 channels of audio sent back from VEPRO, I'm receiving 18mb/s
If I have 24 channels of audio sent back from VEPRO, I'm receiving 42mb/s
I'd have to have about 63/64 tracks streaming to be receiving 100mb/s
